package.json

```diff
@@ -45,14 +46,16 @@
     "schema-utils": "^3.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.12.1",
-    "@babel/core": "^7.12.3",
-    "@babel/preset-env": "^7.12.1",
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
+    "@types/express": "^4.17.8",
+    "@types/jest": "^26.0.15",
+    "@types/mime-types": "^2.1.0",
+    "@types/node": "^10.17.40",
+    "@types/range-parser": "^1.2.3",
```

Comment:
Most of this types should be in `dependencies`

Reply:
Installing @types packages for users is a bad practice that tends to break more than it solves. mime-types, range-parser, and jest are only used internally. @types/node shouldn't be installed for user. what if he wants to verify using @types/node@12 or 14...?